### DISCUSSION TITLE
Welcome to ClassicOS Discussions!

### DISCUSSION THREAD

**Discussion by gbowne1** [gbowne1/ClassicOS#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gbowne1** [comment]
I keep forgetting to welcome everyone who's joined this project or worked on this OS on here 

@Karutoh
@EmptyGroceryBag 
@vmttmv
@shoshta73

And welcome @LilSuperUser

**Reply by LilSuperUser** [reply]
Thanks for the welcome!

I've got some experience from working on the **i8088 OS project**, and I'm looking forward to contributing to this effort to keep 32-bit x86 hardware viable. 

I'll take some time to review the codebase and current discussions to get up to speed. Looking forward to working with everyone here!

**Reply by gbowne1** [reply]
Yes, welcome @LilSuperUser!

We have this discussion and my discord server. Feel free to post bug reports, issues, todos, feature requests, discussions.  Especially report on things that aren't considered correct or things that don't work.

The code here as you see it came from some assumptions I made on how things actually work in 32 bit x86 intel environment.

Yes, with 32 bit support being removed from Linux, I'd like this to be a viable option at some point to replace mainstream 32 bit OS including linux.

**Reply by LilSuperUser** [reply]
Where can I find the link to the discord server?
